lionweb/src/main/kotlin/com/strumenta/kolasu/lionweb/LionWebModelConverter.kt

```diff
@@ -740,11 +747,13 @@ class LionWebModelConverter(
             }
             ParsingResult::class -> {
                 val root = data.getOnlyChildByContainmentName(ParsingResult<*>::root.name)
-                ParsingResult(
+                val tokens = data.getPropertyValue(data.classifier.getPropertyByName("tokens")!!) as TokensList?
```

Comment:
Instead of a string, when possible I use a property reference. Like `ParsingResult::tokens.name`. It helps to find usages and protect against typos

Reply:
I've updated the codebase to use property references here and in other places